awesomemotive/affiliatewp-woocommerce-redirect-affiliates
# Issue/17 - Bump WordPress minimum requirements

Issue #17


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate WooCommerce Redirect Affiliates add-on
2. Confirm the plugin works as expected without any PHP errors.
